src/main/kotlin/org/rust/lang/core/psi/ext/RsAlias.kt

```diff
@@ -5,20 +5,11 @@
 
 package org.rust.lang.core.psi.ext
 
-import com.intellij.lang.ASTNode
-import com.intellij.psi.PsiElement
-import com.intellij.psi.stubs.IStubElementType
 import org.rust.lang.core.psi.RsAlias
-import org.rust.lang.core.stubs.RsAliasStub
+import org.rust.lang.core.psi.RsElementTypes
 
-abstract class RsAliasImplMixin : RsStubbedNamedElementImpl<RsAliasStub>, RsAlias {
-
-    constructor(node: ASTNode) : super(node)
-
-    constructor(stub: RsAliasStub, elementType: IStubElementType<*, *>) : super(stub, elementType)
-
-    override fun getNameIdentifier(): PsiElement? {
-        // `use Foo as _;` really should be unnamed, but "_" is not a valid name in rust, so I think it's ok
-        return identifier ?: underscore
-    }
-}
+val RsAlias.nameLikeText: String
+    get() = greenStub?.nameLikeText
+        ?: node.findChildByType(RsElementTypes.IDENTIFIER)?.text
```

Comment:
Why not just `identifier` instead of `node.findChildByType(RsElementTypes.IDENTIFIER)`?

Reply:
Btw it doesn't support raw identifiers. You should use `unescapedText`